daffodil-test/src/test/scala/org/apache/daffodil/section06/entities/TestEntitiesNew.scala

```diff
@@ -1,32 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements.  See the NOTICE file distributed with
- * this work for additional information regarding copyright ownership.
- * The ASF licenses this file to You under the Apache License, Version 2.0
- * (the "License"); you may not use this file except in compliance with
- * the License.  You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.apache.daffodil.section06.entities
-
-import org.apache.daffodil.util._
-import org.apache.daffodil.tdml.DFDLTestSuite
-
-class TestEntitiesNew {
-  val testDir = "/org/apache/daffodil/section06/entities/"
```

Comment:
What are these *New.scala files that you deleted?  Were they redundant?

Reply:
A long time ago, well before Apache, we had a workflow where developers would add tests to a `FooNew.scala` file and then when a tester verified the test they would move the test to the `Foo.scala` file. I think the idea was that we could easily differentiate between which tests had been verified and which hadn't and track progress. As time went on, testing resources decreased and so we couldn't really maintain this workflow and it didn't really add all that much, so we don't do that anymore. Some of those "New" files still exist, but don't actually have any tests in them because all the tests had been verified and moved out. I'll do another scan for any empty New files and delete them if they're empty. I only noticed these because they used the now deprecated DFDLTestSuite